src/main/groovy/org/checkerframework/gradle/plugin/CheckerFrameworkPlugin.groovy

```diff
@@ -193,24 +199,32 @@ final class CheckerFrameworkPlugin implements Plugin<Project> {
 
       boolean needErrorProneJavac = javaVersion.java8 && isJavac9CF
 
+      def createManifestTask = project.task('createCheckerFrameworkManifest', type: CreateManifestTask)
+      createManifestTask.checkers = userConfig.checkers
 
       project.tasks.withType(AbstractCompile).all { compile ->
         if (compile.hasProperty('options') && (!userConfig.excludeTests || !compile.name.toLowerCase().contains("test"))) {
           compile.options.annotationProcessorPath =
                   compile.options.annotationProcessorPath == null ?
                           project.configurations.checkerFramework :
                           project.configurations.checkerFramework.plus(compile.options.annotationProcessorPath)
-            // Check whether to use the Error Prone javac
-            if (needErrorProneJavac) {
+          // Check whether to use the Error Prone javac
```

Comment:
This indentation change is good, but in general it would be better to put it in a separate commit, to improve understandability both during code review and afterward.

Reply:
I agree, but it's not important enough to delay this PR over.